Devscord.DiscordFramework/WorkflowBuilder.cs

```diff
@@ -21,16 +21,20 @@ public class WorkflowBuilder
         private readonly string _token;
         private readonly IComponentContext _context;
         private readonly Workflow _workflow;
+        private readonly ReactionsController _reactionsController;
 
         private WorkflowBuilder(string token, IComponentContext context, Assembly botAssembly)
         {
-            _client = new DiscordSocketClient(new DiscordSocketConfig
+            this._client = new DiscordSocketClient(new DiscordSocketConfig
             {
                 TotalShards = 1
             });
             this._token = token;
             this._context = context;
             this._workflow = new Workflow(botAssembly, context);
+            this._reactionsController = new ReactionsController();
```

Comment:
to powinno być brane z IComponentContext przez `Resolve<>()`

Reply:
w sensie zamienić na `this._reactionHandler = context.Resolve<ReactionsHandler>();`? bo nie wiem.

Reply:
tak

Reply:
A jaka jest różnica?

Reply:
że bierzesz to z IoC
lepiej mieć definicje jak mają być tworzone obiekty w jednym miejscu
i jak będzie potrzeba zmiany, zmieniać wszystko w jednym miejscu

niż mieć to rozsiane po całej aplikacji